frontend/public/components/nav/navSortUtils.ts

```diff
@@ -87,18 +85,18 @@ export const getSortedNavItems = (navItems: (PluginNavSection | NavItem | Separa
 
 export const sortExtensionItems = (
   extensionItems: (PluginNavSection | NavItem | SeparatorNavItem)[],
-) => {
+): (PluginNavSection | NavItem | SeparatorNavItem)[] => {
```

Comment:
Yeah :love_you_gesture:

Reply:
And if you want make me super happy I think this file could use a type def at the top:

```tsx
import { NavSection as PluginNavSection, NavItem as PluginNavItem, SeparatorNavItem } from '@console/plugin-sdk';

type NavItem = PluginNavSection | PluginNavItem | SeparatorNavItem;
```

And then add types anywhere like:

```tsx
const sortItems = (preSorted: NavItem[], itemsToSort: NavItem[]) => {
```

:smirk_cat:

Reply:
I've trie to make @jerolimov super happy 😉

Reply:
Thanks :)